src/main/java/frc/robot/Robot.java

```diff
@@ -52,6 +61,8 @@ public void robotInit() {
         }
 
         Logger.start();
+        Logger.recordOutput("randomTesting", new Pose2d(0.0, 0.0, new Rotation2d(0.0, 0.0)));
```

Comment:
can we remove this test code before we merge this?

Reply:
Yes